programs/ido-program/migrations/deploy.ts

```diff
@@ -0,0 +1,12 @@
+// Migrations are an early feature. Currently, they're nothing more than this
+// single deploy script that's invoked from the CLI, injecting a provider
+// configured from the workspace's Anchor.toml.
+
+const anchor = require("@project-serum/anchor");
```

Comment:
Update to new package
```suggestion
const anchor = require("@coral-xyz/anchor");
```